easy_profiler_core/CMakeLists.txt

```diff
@@ -233,19 +233,12 @@ if (MSVC)
     target_compile_options(easy_profiler PRIVATE /WX)
 endif ()
 
-if (APPLE)
-    target_compile_options(easy_profiler PUBLIC -std=gnu++11)
-else ()
-    if (CMAKE_VERSION VERSION_LESS "3.1")
-        if (NOT MSVC)
-            target_compile_options(easy_profiler PUBLIC $<$<COMPILE_LANGUAGE:CXX>:-std=gnu++11>)
-        endif ()
-    else ()
-        if (NOT MSVC)
-            target_compile_options(easy_profiler PUBLIC -std=gnu++11)
```

Comment:
I removed this completely because the old L247 property ends up doing exactly the same for you. cmake will add `-std=gnu++11` if the compiler is clang or gcc _for you_.